charts/fybrik/files/openmetadata-connector-conf.yaml

```diff
@@ -0,0 +1,12 @@
+{{- $autoFlag := and .Values.coordinator.enabled (eq .Values.coordinator.catalog "openmetadata") }}
+{{- if include "fybrik.isEnabled" (tuple .Values.openmetadataConnector.enabled $autoFlag) }}
+openmetadata_endpoint: {{ .Values.openmetadataConnector.openmetadata_endpoint | default "openmetadata.fybrik-system:8585/api" }}
+openmetadata_sleep_interval: {{ .Values.openmetadataConnector.openmetadata_sleep_interval | default 100 }}
```

Comment:
1. Why not a key-value config map similarly to opa-connector-cm and others?
2. Should be inside openmetadata-connector directory.

Reply:
Okay. I will move it.
I am using a configmap. See: https://github.com/cdoron/fybrik/blob/openmetadata-connector-templates/charts/fybrik/templates/openmetadata-connector/openmetadata-connector-configmap.yaml#L13

Reply:
Thanks! I will have a look.

Reply:
I moved the contents of `charts/fybrik/files/openmetadata-connector-conf.yaml` into `charts/fybrik/templates/openmetadata-connector/openmetadata-connector-configmap.yaml`.
I hope this is what you had in mind. Thanks

Reply:
No. What I had in mind is something like this:
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: open-metadata-connector-config
data:
  ENDPOINT: {{ .Values.openmetadataConnector.openmetadata_endpoint | default "openmetadata.fybrik-system:8585/api" }}
  SLEEP_INTERVAL: {{ .Values.openmetadataConnector.openmetadata_sleep_interval | default 100 }}
  VAULT_ADDRESS: {{ .Values.openmetadataConnector.vault.address | default "http://vault.fybrik-system:8200"}}
  PRETTY_LOGGING: {{ .Values.global.prettyLogging | quote }}
  LOGGING_VERBOSITY: {{ .Values.global.loggingVerbosity | quote }}
```

Reply:
For now I prefer to continue working with a configuration file.
I will add a task for my to-do list, to move to using environment variables.